scripts-python/xmlquery

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+
+
+"""
+Script to query env_*.xml files.
+If this tool is missing any feature that you need, please notify jgfouca@sandia.gov.
+
+    AUTHOR AND CREDITS
+        Authors: Jim Edwards , .... 
+
+        Based on legacy code with many contributers. (see ...)
+
+    COPYRIGHT AND LICENSE
+    
+        This library is free software; you can redistribute it and/or modify
+        it under the same terms as Python itself.
+"""
+
+
+
+from standard_script_setup import *
+
+from CIME.utils import expect, run_cmd
+from CIME.case import Case
+
+import argparse, doctest, shutil, glob
+import xml.etree.ElementTree as ET
+import textwrap
+
+
+###############################################################################
+def parse_command_line(args, description):
+###############################################################################
+        
+
+    parser = argparse.ArgumentParser(   description="Querry the xml files for attributes and return their values." , 
+                                        formatter_class=argparse.RawDescriptionHelpFormatter, epilog=textwrap.dedent(__doc__) )
+    CIME.utils.setup_standard_logging_options(parser)
+
+    # Set command line options
+    parser.add_argument("attributes", nargs="*"  ,
+                        help="attribute value in env_*.xml file ( <element_name attribute='attribute_value'>value</element_name> )")
+    
+    parser.add_argument("--file", 
```

Comment:
In order to be backward compatible these options should have both single dash and double dash, see file
https://github.com/ESMCI/cime/blob/master/scripts/Tools/xmlchange for an example.